modules/operators/migrations/20_0_0-beta_1-tap-response/index.ts

```diff
@@ -0,0 +1,94 @@
+import { Rule, Tree, SchematicContext } from '@angular-devkit/schematics';
+import {
+  visitTSSourceFiles,
+  createReplaceChange,
+  commitChanges,
+  Change,
+} from '@ngrx/component/schematics-core';
+import { visitCallExpression } from '@ngrx/schematics-core/utility/visitors';
+import * as ts from 'typescript';
+
+export default function migrateTapResponse(): Rule {
+  return (tree: Tree, context: SchematicContext) => {
+    visitTSSourceFiles(tree, (sourceFile: ts.SourceFile) => {
+      const changes: Change[] = [];
+      const visited = new Set<number>();
+      const tapResponseIdentifiers = new Set<string>(['tapResponse']);
+
+      // Track aliases like: const myTapResponse = tapResponse;
+      ts.forEachChild(sourceFile, (node: ts.Node) => {
+        if (ts.isVariableStatement(node)) {
+          for (const decl of node.declarationList.declarations) {
+            if (
```

Comment:
This needs to validate that the import is from `@ngrx/operators`, to be sure we only migrate our signatures and keep the others intact. We can also add a test for this case.